src/main/webapp/availabilityTable.jsp

```diff
@@ -21,7 +21,17 @@
   <tbody>
```

Comment:
lnstead of writing out all the specifics to navigate around the first-slot last-slot situation, can we make a simple for loop? And add data- attributes to the table-success table that have the minTime and maxTime?

Reply:
It won't really be element specific then though. @acmcarther  @jalexanderqed what do you guys think? Prioritize making a nicer jsp page or only put ids on two slots?

Reply:
Fixed it with children accessors.